app/core/ingestion/family.py

```diff
@@ -96,7 +96,7 @@ def _operate_on_family(
             db,
             Family,
             import_id=params.cpr_family_id,
-            extra={**extra, "family_status": FamilyStatus.CREATED},
+            extra={**extra, "family_status": FamilyStatus.PUBLISHED},
```

Comment:
Is there not an issue with setting FamilyStatus=PUBLISHED as there will be a lag time between family creation and finish of the pipeline run? 

I don't think we hit this now as we ingest in staging and copy over but raising whether this could be an issue?

Reply:
Yes it could be- but its the lesser of two evils